services/wordpress/wordpress-downloads.service.js

```diff
@@ -29,13 +29,21 @@ function DownloadsForExtensionType(extensionType) {
       return `Wordpress${capt}Downloads`
     }
 
```

Comment:
Nothing specific to this line, but just want to note that there seems to be a whole lot happening in this file. If I'm not mistaken it ends up generating six different classes?

Could you envision future changes here, such as combining the plugin download intervals?

Reply:
Yea, agreed the download intervals should be combined.

Reply:
👍